src/CovidData/Card/CovidDataInfo.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FunctionComponent } from "react"
+import { useTranslation } from "react-i18next"
+
+import { StyleSheet, View } from "react-native"
+
+import * as CovidData from "../covidData"
+import { Text } from "../../components"
+import LineChart from "../LineChart"
+
+import { Layout, Typography, Spacing } from "../../styles"
+
+interface CovidDataInfoProps {
+  data: CovidData.CovidData
+  locationName: string
+}
+
+const CovidDataInfo: FunctionComponent<CovidDataInfoProps> = ({
+  data,
+  locationName,
+}) => {
+  const { t } = useTranslation()
+
+  const lineData = CovidData.toLineChartCasesPer100k(data)
+
+  const lineChartWidth =
+    0.5 * (Layout.screenWidth - 2 * Spacing.medium - 2 * Spacing.medium)
+  const lineChartHeight = 100
+  return (
+    <View testID={"covid-data"} style={style.container}>
+      <View style={style.header}>
+        <Text style={style.headerText}>
+          {t("covid_data.spread_of_the_virus_in", { locationName })}
+        </Text>
+      </View>
+      <View style={style.dataContainer}>
+        <View style={style.legendContainer}>
+          <Text style={style.legendText}>{"Cases Per 100k"}</Text>
+          <Text style={style.sourceText}>{"source: localcoviddata.org"}</Text>
```

Comment:
this would likely come in a follow up pr, but we should be providing the api source.